package.json

```diff
@@ -10,9 +10,9 @@
     "http": "stream-http"
   },
   "scripts": {
-    "test": "gulp test",
-    "test:node": "gulp test:node",
-    "test:browser": "gulp test:browser",
+    "test": "node --max_old_space_size=4096 node_modules/.bin/gulp test",
+    "test:node": "node --max_old_space_size=4096 node_modules/.bin/gulp test:node",
+    "test:browser": "node --max_old_space_size=4096 node_modules/.bin/gulp test:browser",
```

Comment:
Do we need this? I believe we fixed the webpack hungryness when we migrated all the tests to use factory only